images commit-1

Natrix-Rixtan opened the pull request:
- усі картинки розкладені по папках по секціях
- більшість картинок (крім user у hero) викачані у форматі PNG
- усі картинки продубльовані у форматі WEBP, щоб була можливість задати картинку на прозорому фоні із меншою "вагою"
- картинки із дівчатами у hero та we-will-call-you викачані також (продубльовані) разом із фоном у форматі JPG

Natrix-Rixtan committed:
images commit-1

- усі картинки розкладені по папках по секціях
- більшість картинок (крім user у hero) викачані у форматі PNG
- усі картинки продубльовані у форматі WEBP, щоб була можливість задати картинку на прозорому фоні із меншою "вагою"
- картинки із дівчатами у hero та we-will-call-you викачані також (продубльовані) разом із фоном у форматі JPG

IgorKonoval commented on src/img/hero/hero-girl-dt.png:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/hero/hero-girl-dt.webp:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/hero/hero-girl-dt@2x.png:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/hero/hero-girl-dt@2x.webp:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/hero/hero-girl-mb.png:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/hero/hero-girl-mb.webp:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/hero/hero-girl-mb@2x.png:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/hero/hero-girl-mb@2x.webp:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/hero/hero-girl-tb.png:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/hero/hero-girl-tb.webp:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/hero/hero-girl-tb@2x.png:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/hero/hero-girl-tb@2x.webp:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/modal/modal-girl-tb.webp:
видалити з проекта але залишь на компі (про всяк випадок)
в таблетці така ж картинка як і на десктопі тільки виходить трохи за в'юпорт

IgorKonoval commented on src/img/modal/modal-girl-tb@2x.jpg:
видалити з проекта але залишь на компі (про всяк випадок)
в таблетці така ж картинка як і на десктопі тільки виходить трохи за в'юпорт

IgorKonoval commented on src/img/modal/modal-girl-tb@2x.webp:
видалити з проекта але залишь на компі (про всяк випадок)
в таблетці така ж картинка як і на десктопі тільки виходить трохи за в'юпорт

IgorKonoval commented on src/img/we-will-call-you/call-you-box-dt.png:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/we-will-call-you/call-you-box-dt.webp:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/we-will-call-you/call-you-box-dt@2x.png:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/we-will-call-you/call-you-box-dt@2x.webp:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/we-will-call-you/call-you-box-mb.png:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/we-will-call-you/call-you-box-mb.webp:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/we-will-call-you/call-you-box-mb@2x.png:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/we-will-call-you/call-you-box-mb@2x.webp:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/we-will-call-you/call-you-box-tb.png:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/we-will-call-you/call-you-box-tb.webp:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/we-will-call-you/call-you-box-tb@2x.png:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval commented on src/img/we-will-call-you/call-you-box-tb@2x.webp:
видалити з проекта але залишь на компі (про всяк випадок)

IgorKonoval reviewed (changes requested):
Видали непотрібні файли.
Добав файлик в дирочках

Natrix-Rixtan committed:
images commit-2

- видалені вказані файли
- додано файли фону із крапочками для секцій how-it-works i we-will-call-you

IgorKonoval commented on src/img/we-will-call-you/call-you-background.png:
вделай его один и кинь в корень, т.к. он используется в 2-х секциях

IgorKonoval reviewed (changes requested):
добавить  планшетную и мобильную версии BG

Natrix-Rixtan committed:
images commit-3

- додано файли бекграунду 2x та tablet + mobile версій
